#! /bin/sh /usr/share/dpatch/dpatch-run
## 01-perl-she-bang.dpatch by Thomas Girard <thomas.g.girard@free.fr>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Add #!/usr/bin/perl where needed.

@DPATCH@

Index: ACE_wrappers/TAO/CIAO/examples/Hello/descriptors/run_test_default.pl
===================================================================
--- ACE_wrappers.orig/TAO/CIAO/examples/Hello/descriptors/run_test_default.pl	1970-05-16 13:42:32.000000000 +0100
+++ ACE_wrappers/TAO/CIAO/examples/Hello/descriptors/run_test_default.pl	2004-10-26 13:20:18.398300424 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+#
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/examples/IPC_SAP/SSL_SAP/summarize
===================================================================
--- ACE_wrappers.orig/examples/IPC_SAP/SSL_SAP/summarize	1999-11-23 02:59:48.000000000 +0100
+++ ACE_wrappers/examples/IPC_SAP/SSL_SAP/summarize	2004-10-26 13:20:18.410298600 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -w -S $0 ${1+"$@"}'
     & eval 'exec perl -w -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/POA/Adapter_Activator/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/POA/Adapter_Activator/run_test.pl	2002-08-22 20:08:03.000000000 +0200
+++ ACE_wrappers/TAO/examples/POA/Adapter_Activator/run_test.pl	2004-10-26 13:20:18.410298600 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/PluggableUDP/tests/SimplePerformance/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/PluggableUDP/tests/SimplePerformance/run_test.pl	2001-07-17 05:27:15.000000000 +0200
+++ ACE_wrappers/TAO/examples/PluggableUDP/tests/SimplePerformance/run_test.pl	2004-10-26 13:20:18.410298600 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/Simple/time-date/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/Simple/time-date/run_test.pl	2002-10-21 21:54:38.000000000 +0200
+++ ACE_wrappers/TAO/examples/Simple/time-date/run_test.pl	2004-10-26 13:20:18.411298448 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/examples/IPC_SAP/SOCK_SAP/summarize
===================================================================
--- ACE_wrappers.orig/examples/IPC_SAP/SOCK_SAP/summarize	1998-10-09 23:27:48.000000000 +0200
+++ ACE_wrappers/examples/IPC_SAP/SOCK_SAP/summarize	2004-10-26 13:20:18.417297536 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -w -S $0 ${1+"$@"}'
     & eval 'exec perl -w -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/RTCORBA/Activity/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/RTCORBA/Activity/run_test.pl	2003-07-24 20:18:31.000000000 +0200
+++ ACE_wrappers/TAO/examples/RTCORBA/Activity/run_test.pl	2004-10-26 13:20:18.418297384 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/RTScheduling/Fixed_Priority_Scheduler/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/RTScheduling/Fixed_Priority_Scheduler/run_test.pl	2003-08-12 23:50:39.000000000 +0200
+++ ACE_wrappers/TAO/examples/RTScheduling/Fixed_Priority_Scheduler/run_test.pl	2004-10-26 13:20:18.418297384 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+#
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/RTScheduling/MIF_Scheduler/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/RTScheduling/MIF_Scheduler/run_test.pl	2003-08-12 23:51:05.000000000 +0200
+++ ACE_wrappers/TAO/examples/RTScheduling/MIF_Scheduler/run_test.pl	2004-10-26 13:20:18.418297384 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+#
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/AMH/Sink_Server/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/AMH/Sink_Server/run_test.pl	2002-10-21 21:54:38.000000000 +0200
+++ ACE_wrappers/TAO/examples/AMH/Sink_Server/run_test.pl	2004-10-26 13:20:18.418297384 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/POA/On_Demand_Loading/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/POA/On_Demand_Loading/run_test.pl	2002-08-22 20:08:04.000000000 +0200
+++ ACE_wrappers/TAO/examples/POA/On_Demand_Loading/run_test.pl	2004-10-26 13:20:18.418297384 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/POA/NewPOA/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/POA/NewPOA/run_test.pl	2001-03-02 03:31:02.000000000 +0100
+++ ACE_wrappers/TAO/examples/POA/NewPOA/run_test.pl	2004-10-26 13:20:18.419297232 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/PluggableUDP/tests/Basic/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/PluggableUDP/tests/Basic/run_test.pl	1970-06-07 12:24:47.000000000 +0100
+++ ACE_wrappers/TAO/examples/PluggableUDP/tests/Basic/run_test.pl	2004-10-26 13:20:18.419297232 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/examples/QOS/Diffserv/run_test.pl
===================================================================
--- ACE_wrappers.orig/examples/QOS/Diffserv/run_test.pl	2001-11-22 11:30:55.000000000 +0100
+++ ACE_wrappers/examples/QOS/Diffserv/run_test.pl	2004-10-26 13:20:18.433295104 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/POA/FindPOA/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/POA/FindPOA/run_test.pl	2001-03-02 03:31:02.000000000 +0100
+++ ACE_wrappers/TAO/examples/POA/FindPOA/run_test.pl	2004-10-26 13:20:18.433295104 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/Buffered_Oneways/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/Buffered_Oneways/run_test.pl	2001-04-24 10:02:41.000000000 +0200
+++ ACE_wrappers/TAO/examples/Buffered_Oneways/run_test.pl	2004-10-26 13:20:18.433295104 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/Quoter/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/Quoter/run_test.pl	2001-11-05 20:13:43.000000000 +0100
+++ ACE_wrappers/TAO/examples/Quoter/run_test.pl	2004-10-26 13:20:18.433295104 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/Advanced/ch_21/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/Advanced/ch_21/run_test.pl	1999-09-04 00:49:05.000000000 +0200
+++ ACE_wrappers/TAO/examples/Advanced/ch_21/run_test.pl	2004-10-26 13:20:18.434294952 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/Load_Balancing/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/Load_Balancing/run_test.pl	2001-11-05 20:13:42.000000000 +0100
+++ ACE_wrappers/TAO/examples/Load_Balancing/run_test.pl	2004-10-26 13:20:18.434294952 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/Simple/echo/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/Simple/echo/run_test.pl	2001-03-05 04:23:46.000000000 +0100
+++ ACE_wrappers/TAO/examples/Simple/echo/run_test.pl	2004-10-26 13:20:18.434294952 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/Logging/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/Logging/run_test.pl	2001-03-05 04:23:46.000000000 +0100
+++ ACE_wrappers/TAO/examples/Logging/run_test.pl	2004-10-26 13:20:18.434294952 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/Advanced/ch_8_and_10/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/Advanced/ch_8_and_10/run_test.pl	1999-08-24 23:35:33.000000000 +0200
+++ ACE_wrappers/TAO/examples/Advanced/ch_8_and_10/run_test.pl	2004-10-26 13:20:18.434294952 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/POA/POA_BiDir/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/POA/POA_BiDir/run_test.pl	2002-04-06 23:29:03.000000000 +0200
+++ ACE_wrappers/TAO/examples/POA/POA_BiDir/run_test.pl	2004-10-26 13:20:18.435294800 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/POA/RootPOA/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/POA/RootPOA/run_test.pl	2001-03-02 03:31:02.000000000 +0100
+++ ACE_wrappers/TAO/examples/POA/RootPOA/run_test.pl	2004-10-26 13:20:18.435294800 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/POA/Reference_Counted_Servant/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/POA/Reference_Counted_Servant/run_test.pl	2002-08-22 20:08:04.000000000 +0200
+++ ACE_wrappers/TAO/examples/POA/Reference_Counted_Servant/run_test.pl	2004-10-26 13:20:18.435294800 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/POA/Forwarding/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/POA/Forwarding/run_test.pl	2001-03-05 04:23:46.000000000 +0100
+++ ACE_wrappers/TAO/examples/POA/Forwarding/run_test.pl	2004-10-26 13:20:18.435294800 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/bin/ace_ld
===================================================================
--- ACE_wrappers.orig/bin/ace_ld	2002-12-23 13:00:35.000000000 +0100
+++ ACE_wrappers/bin/ace_ld	2004-10-26 13:20:18.446293128 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/Simple/grid/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/Simple/grid/run_test.pl	2001-03-05 04:23:46.000000000 +0100
+++ ACE_wrappers/TAO/examples/Simple/grid/run_test.pl	2004-10-26 13:20:18.446293128 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/ior_corbaloc/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/ior_corbaloc/run_test.pl	2002-12-27 20:51:37.000000000 +0100
+++ ACE_wrappers/TAO/examples/ior_corbaloc/run_test.pl	2004-10-26 13:20:18.446293128 +0200
@@ -1,4 +1,6 @@
-     eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
+#!/usr/bin/perl
+
+eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
   &  eval 'exec perl -S $0 $argv:q'
   if 0;
 
Index: ACE_wrappers/TAO/examples/POA/On_Demand_Activation/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/POA/On_Demand_Activation/run_test.pl	2002-08-22 20:08:04.000000000 +0200
+++ ACE_wrappers/TAO/examples/POA/On_Demand_Activation/run_test.pl	2004-10-26 13:20:18.447292976 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/OBV/Typed_Events/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/OBV/Typed_Events/run_test.pl	2001-03-05 04:23:46.000000000 +0100
+++ ACE_wrappers/TAO/examples/OBV/Typed_Events/run_test.pl	2004-10-26 13:20:18.447292976 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/Persistent_Grid/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/Persistent_Grid/run_test.pl	2001-03-05 04:23:46.000000000 +0100
+++ ACE_wrappers/TAO/examples/Persistent_Grid/run_test.pl	2004-10-26 13:20:18.447292976 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/POA/Explicit_Activation/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/POA/Explicit_Activation/run_test.pl	2002-08-22 20:08:03.000000000 +0200
+++ ACE_wrappers/TAO/examples/POA/Explicit_Activation/run_test.pl	2004-10-26 13:20:18.447292976 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/Buffered_AMI/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/Buffered_AMI/run_test.pl	2001-04-24 10:02:41.000000000 +0200
+++ ACE_wrappers/TAO/examples/Buffered_AMI/run_test.pl	2004-10-26 13:20:18.447292976 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/Advanced/ch_18/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/Advanced/ch_18/run_test.pl	1999-08-24 23:35:23.000000000 +0200
+++ ACE_wrappers/TAO/examples/Advanced/ch_18/run_test.pl	2004-10-26 13:20:18.448292824 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/TypeCode_Creation/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/TypeCode_Creation/run_test.pl	2001-04-03 03:24:26.000000000 +0200
+++ ACE_wrappers/TAO/examples/TypeCode_Creation/run_test.pl	2004-10-26 13:20:18.448292824 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/CIAO/examples/Hello/descriptors/run_test_remote.pl
===================================================================
--- ACE_wrappers.orig/TAO/CIAO/examples/Hello/descriptors/run_test_remote.pl	1970-05-16 13:42:32.000000000 +0100
+++ ACE_wrappers/TAO/CIAO/examples/Hello/descriptors/run_test_remote.pl	2004-10-26 13:20:18.448292824 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+#
 #run_test_remote.pl,v 1.2 2004/05/25 02:19:36 lut Exp
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
Index: ACE_wrappers/TAO/examples/Advanced/ch_12/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/Advanced/ch_12/run_test.pl	1999-08-24 23:35:18.000000000 +0200
+++ ACE_wrappers/TAO/examples/Advanced/ch_12/run_test.pl	2004-10-26 13:20:18.448292824 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/Simple/time/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/Simple/time/run_test.pl	2001-11-05 20:13:43.000000000 +0100
+++ ACE_wrappers/TAO/examples/Simple/time/run_test.pl	2004-10-26 13:20:18.448292824 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/Advanced/ch_3/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/Advanced/ch_3/run_test.pl	1999-08-24 23:35:30.000000000 +0200
+++ ACE_wrappers/TAO/examples/Advanced/ch_3/run_test.pl	2004-10-26 13:20:18.449292672 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/examples/Reactor/WFMO_Reactor/run_test.pl
===================================================================
--- ACE_wrappers.orig/examples/Reactor/WFMO_Reactor/run_test.pl	1970-03-01 01:53:51.000000000 +0100
+++ ACE_wrappers/examples/Reactor/WFMO_Reactor/run_test.pl	2004-10-26 13:20:18.461290848 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/POA/TIE/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/POA/TIE/run_test.pl	2001-03-02 03:31:02.000000000 +0100
+++ ACE_wrappers/TAO/examples/POA/TIE/run_test.pl	2004-10-26 13:20:18.461290848 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/Advanced/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/Advanced/run_test.pl	1999-08-24 23:35:13.000000000 +0200
+++ ACE_wrappers/TAO/examples/Advanced/run_test.pl	2004-10-26 13:20:18.461290848 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/Simple/bank/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/Simple/bank/run_test.pl	2001-03-05 04:23:46.000000000 +0100
+++ ACE_wrappers/TAO/examples/Simple/bank/run_test.pl	2004-10-26 13:20:18.461290848 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/POA/Default_Servant/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/POA/Default_Servant/run_test.pl	2001-03-02 03:31:02.000000000 +0100
+++ ACE_wrappers/TAO/examples/POA/Default_Servant/run_test.pl	2004-10-26 13:20:18.461290848 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/Callback_Quoter/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/Callback_Quoter/run_test.pl	2001-03-05 04:23:46.000000000 +0100
+++ ACE_wrappers/TAO/examples/Callback_Quoter/run_test.pl	2004-10-26 13:20:18.462290696 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/PluggableUDP/tests/Performance/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/PluggableUDP/tests/Performance/run_test.pl	2001-04-21 07:46:00.000000000 +0200
+++ ACE_wrappers/TAO/examples/PluggableUDP/tests/Performance/run_test.pl	2004-10-26 13:20:18.462290696 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/POA/Loader/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/POA/Loader/run_test.pl	2002-08-22 20:08:04.000000000 +0200
+++ ACE_wrappers/TAO/examples/POA/Loader/run_test.pl	2004-10-26 13:20:18.462290696 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/Simulator/DOVEBrowser/ecdata_patch.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/Simulator/DOVEBrowser/ecdata_patch.pl	2000-05-24 18:45:19.000000000 +0200
+++ ACE_wrappers/TAO/examples/Simulator/DOVEBrowser/ecdata_patch.pl	2004-10-26 13:20:18.462290696 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -pi -S $0 ${1+"$@"}'
     & eval 'exec perl -pi -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/examples/Export/run_test.pl
===================================================================
--- ACE_wrappers.orig/examples/Export/run_test.pl	2001-03-05 04:40:39.000000000 +0100
+++ ACE_wrappers/examples/Export/run_test.pl	2004-10-26 13:20:18.474288872 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/examples/POA/DSI/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/examples/POA/DSI/run_test.pl	2001-11-05 20:13:42.000000000 +0100
+++ ACE_wrappers/TAO/examples/POA/DSI/run_test.pl	2004-10-26 13:20:18.475288720 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/orbsvcs/examples/RtEC/IIOPGateway/consumerec_crash.pl
===================================================================
--- ACE_wrappers.orig/TAO/orbsvcs/examples/RtEC/IIOPGateway/consumerec_crash.pl	2003-08-15 12:05:14.000000000 +0200
+++ ACE_wrappers/TAO/orbsvcs/examples/RtEC/IIOPGateway/consumerec_crash.pl	2004-10-26 13:20:18.475288720 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+#
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/orbsvcs/examples/Notify/Filter/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/orbsvcs/examples/Notify/Filter/run_test.pl	2003-07-06 23:41:58.000000000 +0200
+++ ACE_wrappers/TAO/orbsvcs/examples/Notify/Filter/run_test.pl	2004-10-26 13:20:18.475288720 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+#
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/orbsvcs/examples/Notify/ThreadPool/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/orbsvcs/examples/Notify/ThreadPool/run_test.pl	2003-07-24 23:38:36.000000000 +0200
+++ ACE_wrappers/TAO/orbsvcs/examples/Notify/ThreadPool/run_test.pl	2004-10-26 13:20:18.475288720 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/perl
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/orbsvcs/examples/RtEC/IIOPGateway/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/orbsvcs/examples/RtEC/IIOPGateway/run_test.pl	2003-08-15 12:05:14.000000000 +0200
+++ ACE_wrappers/TAO/orbsvcs/examples/RtEC/IIOPGateway/run_test.pl	2004-10-26 13:20:18.476288568 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+#
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/orbsvcs/examples/LoadBalancing/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/orbsvcs/examples/LoadBalancing/run_test.pl	2003-10-30 00:39:36.000000000 +0100
+++ ACE_wrappers/TAO/orbsvcs/examples/LoadBalancing/run_test.pl	2004-10-26 13:20:18.476288568 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+#
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/orbsvcs/examples/CosEC/RtEC_Based/tests/Multiple/run_test2.pl
===================================================================
--- ACE_wrappers.orig/TAO/orbsvcs/examples/CosEC/RtEC_Based/tests/Multiple/run_test2.pl	2001-03-05 10:51:13.000000000 +0100
+++ ACE_wrappers/TAO/orbsvcs/examples/CosEC/RtEC_Based/tests/Multiple/run_test2.pl	2004-10-26 13:20:18.476288568 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/orbsvcs/examples/Notify/Subscribe/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/orbsvcs/examples/Notify/Subscribe/run_test.pl	2003-07-06 23:41:58.000000000 +0200
+++ ACE_wrappers/TAO/orbsvcs/examples/Notify/Subscribe/run_test.pl	2004-10-26 13:20:18.476288568 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+#
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/orbsvcs/examples/Notify/Lanes/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/orbsvcs/examples/Notify/Lanes/run_test.pl	2003-07-24 23:38:36.000000000 +0200
+++ ACE_wrappers/TAO/orbsvcs/examples/Notify/Lanes/run_test.pl	2004-10-26 13:20:18.476288568 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+#
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/orbsvcs/examples/CosEC/Simple/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/orbsvcs/examples/CosEC/Simple/run_test.pl	2001-03-05 10:51:13.000000000 +0100
+++ ACE_wrappers/TAO/orbsvcs/examples/CosEC/Simple/run_test.pl	2004-10-26 13:20:18.477288416 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/orbsvcs/examples/Security/Send_File/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/orbsvcs/examples/Security/Send_File/run_test.pl	2002-10-21 21:54:38.000000000 +0200
+++ ACE_wrappers/TAO/orbsvcs/examples/Security/Send_File/run_test.pl	2004-10-26 13:20:18.477288416 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/orbsvcs/examples/CosEC/RtEC_Based/tests/Multiple/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/orbsvcs/examples/CosEC/RtEC_Based/tests/Multiple/run_test.pl	2001-03-05 10:51:13.000000000 +0100
+++ ACE_wrappers/TAO/orbsvcs/examples/CosEC/RtEC_Based/tests/Multiple/run_test.pl	2004-10-26 13:20:18.477288416 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/orbsvcs/examples/RtEC/Simple/run_test.pl
===================================================================
--- ACE_wrappers.orig/TAO/orbsvcs/examples/RtEC/Simple/run_test.pl	2003-04-09 19:14:22.000000000 +0200
+++ ACE_wrappers/TAO/orbsvcs/examples/RtEC/Simple/run_test.pl	2004-10-26 13:20:18.477288416 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/bin/mwc.pl
===================================================================
--- ACE_wrappers.orig/bin/mwc.pl	1970-05-10 01:47:12.000000000 +0100
+++ ACE_wrappers/bin/mwc.pl	2004-10-26 13:20:18.484287352 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+#
 eval '(exit $?0)' && eval 'exec perl -w -S $0 ${1+"$@"}'
     & eval 'exec perl -w -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/bin/mpc.pl
===================================================================
--- ACE_wrappers.orig/bin/mpc.pl	1970-05-10 01:47:12.000000000 +0100
+++ ACE_wrappers/bin/mpc.pl	2004-10-26 13:20:18.495285680 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+#
 eval '(exit $?0)' && eval 'exec perl -w -S $0 ${1+"$@"}'
     & eval 'exec perl -w -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/MPC/prj_install.pl
===================================================================
--- ACE_wrappers.orig/MPC/prj_install.pl	1970-05-03 03:17:37.000000000 +0100
+++ ACE_wrappers/MPC/prj_install.pl	2004-10-26 13:20:40.621921920 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+#
 eval '(exit $?0)' && eval 'exec perl -w -S $0 ${1+"$@"}'
     & eval 'exec perl -w -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/MPC/mwc.pl
===================================================================
--- ACE_wrappers.orig/MPC/mwc.pl	1970-05-10 02:02:37.000000000 +0100
+++ ACE_wrappers/MPC/mwc.pl	2004-10-26 13:20:40.639919184 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+#
 eval '(exit $?0)' && eval 'exec perl -w -S $0 ${1+"$@"}'
     & eval 'exec perl -w -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/MPC/clone_build_tree.pl
===================================================================
--- ACE_wrappers.orig/MPC/clone_build_tree.pl	1970-07-20 01:39:35.000000000 +0100
+++ ACE_wrappers/MPC/clone_build_tree.pl	2004-10-26 13:20:40.639919184 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+#
 eval '(exit $?0)' && eval 'exec perl -w -S $0 ${1+"$@"}'
     & eval 'exec perl -w -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/CIAO/examples/Hello/descriptors/run_test_debug.pl
===================================================================
--- ACE_wrappers.orig/TAO/CIAO/examples/Hello/descriptors/run_test_debug.pl	1970-02-02 04:51:28.000000000 +0100
+++ ACE_wrappers/TAO/CIAO/examples/Hello/descriptors/run_test_debug.pl	2004-10-26 13:20:40.622921768 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+#
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/MPC/mpc.pl
===================================================================
--- ACE_wrappers.orig/MPC/mpc.pl	1970-05-10 02:02:37.000000000 +0100
+++ ACE_wrappers/MPC/mpc.pl	2004-10-26 13:20:40.640919032 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+#
 eval '(exit $?0)' && eval 'exec perl -w -S $0 ${1+"$@"}'
     & eval 'exec perl -w -S $0 $argv:q'
     if 0;
Index: ACE_wrappers/TAO/CIAO/examples/Hello/descriptors/run_test_multirecv.pl
===================================================================
--- ACE_wrappers.orig/TAO/CIAO/examples/Hello/descriptors/run_test_multirecv.pl	1970-01-20 05:59:30.000000000 +0100
+++ ACE_wrappers/TAO/CIAO/examples/Hello/descriptors/run_test_multirecv.pl	2004-10-26 13:20:40.622921768 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/perl
+#
 eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
     & eval 'exec perl -S $0 $argv:q'
     if 0;

